.travis.yml

```diff
@@ -4,18 +4,19 @@ jdk: openjdk11
 
 scala:
   - 2.12.10
+  - 2.13.1
 
 branches:
   only:
     - master
 
 cache:
   directories:
-    - "$HOME/.ivy2"
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/boot
```

Comment:
Maybe change this to? Apparently this is better for caching.
```
# copied from https://github.com/fthomas/scala-steward/blob/7f260bc8789381a2754dea00d4df557e8d6dc785/.travis.yml
before_cache:
  - find $HOME/.ivy2/cache -name "ivydata-*.properties" -type f -delete
  - find $HOME/.sbt -name "*.lock" -type f -delete
cache:
  directories:
    - $HOME/google-cloud-sdk
    - $HOME/.cache/coursier
    - $HOME/.ivy2/cache
    - $HOME/.sbt
```

Reply:
I think the `before_cache` stuff might fail on sbt 1.3? Trying now.

Reply:
(It worked, so I'm leaving it.)